src/test/util/test-rig-command.ts

```diff
@@ -140,6 +142,22 @@ export class WireitTestRigCommandInvocation {
     this.command = command;
   }
 
+  async environment(): Promise<Exclude<EnvironmentResponseMessage, 'type'>> {
+    this.#assertState('connected');
+    const client = new IpcClient<ChildToRigMessage, RigToChildMessage>(
+      this.#socket
+    );
+    client.send({type: 'environmentRequest'});
+    for await (const message of client.receive()) {
+      switch (message.type) {
```

Comment:
Hm, this silently discards all other messages until an environmentResponse is received

I wonder if this class should extend IpcClient, override the receive method, and pass along everything but 'environmentResponse' which it handles by matching up with an environmentRequest

Reply:
Done, or something like it.